GANDLF/metrics/synthesis.py

```diff
@@ -39,15 +39,21 @@ def structural_similarity_index(
     return ssim_idx.mean()
 
 
-def mean_squared_error(prediction: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
+def mean_squared_error(
```

Comment:
This is my personal preference, but what about separating this case into `mean_squared_error` and `root_mean_squared` error, wtih `squared` flag set properly for each of them. I found this flag in pytorch api some time ago and it was puzzling to me why they did not separate those into MSE and RMSE, it is much easier for the reader to see the difference purely by it's name.
If we stay here, I would directly write that it `squared` flag set turns it into MSE, and edit return docstring accordingly.